Switch site-wide form to Mailchimp form

alex-novak opened the pull request:
https://trello.com/c/QI4MSGPu

Commit:
[CPW-66] Switch footer form to Mailchimp, add form to Partner page

albertvolkman commented:
I think we could include this in the layout file instead of including it on each template. Also, I'm assuming this form will be used on the DrupalContractors site in the future, so I believe this should be added to the esteemed-theme repository instead of this one. @mpritchardESTM will both sites utilize the same form?

mpritchardESTM commented:
Yes, we will use the same form

alex-novak commented:
> I think we could include this in the layout file instead of including it on each template. Also, I'm assuming this form will be used on the DrupalContractors site in the future, so I believe this should be added to the esteemed-theme repository instead of this one. @mpritchardESTM will both sites utilize the same form?

1)
Removed form template
Added this form to footer layout
add default flag to _config.yml
removed all includes from all templates
exclude this form by flag on needed pages

2) we already in esteemed-theme repository
